lib/actions/helpResponse.js

```diff
@@ -0,0 +1,9 @@
+class HelpResponse {
+  /* eslint-disable-next-line class-methods-use-this */
+  sendToBot(bot, message) {
+    return bot.reply(message,
+      'To add a link: /project add label link \n To Delete a Link: /project delete label \n To view all links: /project \n For help: /project help');
```

Comment:
Wrap the variable parameters in square brackets to indicate they are variables, or use example data.

```
To add a link: /project add [label] [url]
To add a link: /project add Jira https://sb.jira.com
```